client/src/AddToList.jsx

```diff
@@ -25,19 +27,28 @@ class AddToList extends React.Component {
   }
 
   render() {
+    const styleArrowButton = {
+      background: 'url(./images/croppedListArrow.png)',
```

Comment:
i wasn't able to get local images to load in my component! do you mind showing me how?